loops.js

```diff
@@ -33,6 +42,13 @@
 // Tip: een getal is oneven als je het door 2 deelt en er blijft nog iets over. Een getal is even als je het deelt door 2 en er blijft niets over. Google daarom eens naar 'mdn webdocs remainder operator'.
 // ==========================================
 
+for(let i = 0; i<15; i++) {
```

Comment:
Volgens de uitwerkingen is de conditie binnen de for-loop i < 16
Bij jou staat i < 15
Als de conditie i < 15 is, dan loopt volgens mij de loop tot 14, en niet tot 15


for (let i = 0; i < 16; i++) {
  if (i % 2 === 0) {
    console.log(i + ' is even');
  }
  if (i % 2 !== 0) {
    console.log(i + ' is oneven');
  }
}


for (let i = 0; i < 16; i++) {
  if (i % 2 === 0) {
    console.log(i + ' is even');
  }
  if (i % 2 !== 0) {
    console.log(i + ' is oneven');
  }
}